packages/react-dom/src/client/ReactDOM.js

```diff
@@ -1125,6 +1126,7 @@ function createPortal(
     isValidContainer(container),
     'Target container is not a DOM element.',
   );
+  trapClickOnNonInteractiveElement(((container: any): HTMLElement));
```

Comment:
Hmm, not sure we want to introduce side effects there.

Reply:
Is there another place to perform DOM-specific effects when a portal is mounted?

Reply:
Something like `appendToContainer` in the renderer?

Reply:
Hmm maybe? Idk :-(